docs/tasks.md

```diff
@@ -165,6 +165,10 @@ Write a new stack for the app publishing pipeline.
 * A maximum of 2000 points will be awarded to each participant.
 * Open a pull request in this repository to notify testnet coordinators you have developed a new working stack.
 
+### Celestia-node stack
+- Code: https://github.com/BlackBlocks-io/stack-orchestrator/commit/e8b91e601b41192f30733f1b3f60c3fc7eb3e47b
+- Demo: https://github.com/BlackBlocks-io/stack-orchestrator/commit/e8b91e601b41192f30733f1b3f60c3fc7eb3e47b#commitcomment-145918265
```

Comment:
nice. FYI, a stack definition should be a standalone repo and gotten with the `--fetch-stack` flag, eg., see https://git.vdb.to/cerc-io/stack-orchestrator/issues/934